addons/website/data/website_contactus_data.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="website.contactus" name="Contact Us">
+        <t t-call="website.layout">
+            <span class="hidden" data-for="contactus_form" t-att-data-values="{'email_to': res_company.email}" />
+            <div id="wrap" class="oe_structure">
+                <section class="s_title parallax s_parallax_is_fixed bg-black-50 pt24 pb24" data-vcss="001" data-snippet="s_title" data-scroll-background-ratio="1">
+                    <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_banner_default_image'); background-position: 50% 0;" />
+                    <div class="o_we_bg_filter bg-black-50" />
+                    <div class="container">
+                        <h1>Contact us</h1>
+                    </div>
+                </section>
+                <section class="s_text_block pt40 pb40 o_colored_level " data-snippet="s_text_block">
+                    <div class="s_allow_columns container mt-4 mb-5">
```

Comment:
remove the `mt-4 mb-5`, this is creating a double spacing as there is already a spacing on the line above (section pt40 & pb40) + margin can't be changed by editor